stvs/SageTV7/SageTV7.xml

```diff
@@ -67182,7 +67182,7 @@
           </Action>
           <Action Name="&quot;REM Option for: Configure Placeshifter Connection&quot;" Sym="OPUS4A-210962">
            <Conditional Name="!IsEmbeddedSystem()" Sym="OPUS4A-211270">
-            <Conditional Name="( IsServerUI() || (!IsDesktopUI() &amp;&amp; IsRemoteUI()) || (IsLinuxOS() &amp;&amp; GetServerProperty(&quot;linux/enable_nas&quot;,false)) ) &amp;&amp; IsSageTVServerEnabled()" Sym="BASE-45755">
+            <Conditional Name="IsLocalConnection() &amp;&amp; IsSageTVServerEnabled()" Sym="BASE-45755">
```

Comment:
Okay, I can fix that I think by checking:
`( IsServerUI() || ( IsRemoteUI() && IsLocalConnection() ) ) && IsSageTVServerEnabled()`

I noticed the CLA check isn't working - it appears github isn't associating the commits from my newer dev machine with my CLA's login.  And tips on that?